engine/src/lin_time.cpp

```diff
@@ -149,7 +150,8 @@ void micro_sleep( unsigned int n )
 
 #elif defined (IRIX)
 
-#include <unistd.h>
+// Now included above, for all operating systems except Windows
+// #include <unistd.h>
```

Comment:
yes we need to clean a lot of this up; feel free to just drop it out entirely.

NOTE: FTR Only time a header should be included lower in a file is if there's a define that changes it as a result, even then we should try to keep as much of that at the top as possible.

Reply:
Dropped entirely.